File: Android_RISCC/app/src/main/java/com/cronelab/riscc/support/common/validation/Validation.kt
package com.cronelab.riscc.support.common.validation


class Validation {
    var message: String
    var status: Boolean

    constructor() {
        message = "NA"
        status = false
    }

    constructor(message: String, status: Boolean) {
        this.message = message
        this.status = status
    }
}

File: iOS_RISCC/Model/Validation.swift
//
//  Validation.swift
//  RISCC
//
//  Created by Jeevan Chaudhary on 18/03/2021.
//

import UIKit

class Validation: NSObject {

    var status:Bool
    var message:String
    
    override init() {
        self.status = false
        self.message = ""
    }
    
    init(status:Bool, message:String) {
        self.status = status
        self.message = message
    }
}
